var/spack/repos/builtin/packages/r-pegas/package.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class RPegas(RPackage):
+    """Functions for reading, writing, plotting, analysing, and
```

Comment:
Spack supports both a short description and a long description, with the short description being the text up to a blank line, followed by the long description. For R packages that can be achieved with using the `Title:` field as the short description and the `Description:` field as the long description. 

With that in mind, please add the title of the package as the first line in the text blurb.

```
"""Population and Evolutionary Genetics Analysis System

Functions for reading, writing, plotting, analysing, and
..."""